pkg/iac/scanners/terraformplan/tfjson/test/scanner_test.go

```diff
@@ -33,7 +33,7 @@ func Test_Scanning_Plan(t *testing.T) {
 			failedResults = append(failedResults, r)
 		}
 	}
-	assert.Len(t, results, 15)
```

Comment:
Rego check is [added](https://github.com/aquasecurity/trivy/blob/main/pkg/iac/rego/scanner.go#L415) successfully passed when no misconfigs are detected using it, unlike Go check where it was done [manually](https://github.com/aquasecurity/trivy-checks/blob/main/checks/cloud/aws/accessanalyzer/enable_access_analyzer.go#L54).